packages/components/src/ProgressBar/_docs/ProgressBar.stickersheet.stories.tsx

```diff
@@ -25,57 +25,94 @@ export default {
   },
 } satisfies Meta
 
+const moods = [
```

Comment:
Just as a suggestion, but in some of my other ones I just had this as an array of the values and just let the title be lowercase. Saves a bunch of lines, and it's inconsequential given it's only for testing. But up to you :)

Reply:
True, probably not worth applying that here, but next time